### ISSUE TITLE
Mouse input not working in XWayland windows on secondary outputs

### ISSUE THREAD

**Issue by Soulusions** [WayfireWM/wayfire#1718]
**Describe the bug**
When XWayland windows (tested vivaldi, discord, vscode) are on a secondary output rather than the primary, they are unable to receive mouse input (keyboard navigation and other non-mouse related features still work just fine) 

**To Reproduce**
1. Have 2 outputs (I am not able to try with more)
2. Open XWayland application on the secondary output (or move it there from primary)
3. Mouse input should be impossible with the window now

**Expected behavior**
Mouse input should work on both primary and secondary display outputs.

**Screenshots or stacktrace**
My system is currently not set up for multi-window recording so I cannot provide video, which shouldn't be a problem since the problem is pretty simple, but if it becomes necessary i'll try to provide it later!

**Wayfire version**
0.7.5, git

**Comment by ammen99** [comment]
Do some of your outputs perhaps have negative coordinates?

**Comment by Soulusions** [comment]
Ah yes!
My secondary output being located above my primary I think it has a negative Y

**Comment by ammen99** [comment]
Xwayland does not support negative output positions, whereas Wayland does. You should reconfigure your outputs so that they all have positive coordinates, and then hopefully everything will work.
